samples/chatbot-easy-rag-kotlin/README.md

```diff
@@ -0,0 +1,73 @@
+# Financial Assistant Chatbot with Easy RAG
+
+This example demonstrates how to create a financial assistant chatbot with Retrieval Augmented Generation (RAG) using
+`quarkus-langchain4j` and Kotlin, specifically utilizing the Easy RAG extension.
+For more information about Easy RAG, refer to the file 
+`docs/modules/ROOT/pages/easy-rag.adoc`.
+
+## Running the example
+
+A prerequisite to running this example is to provide your OpenAI API key.
+
+```
+export QUARKUS_LANGCHAIN4J_OPENAI_API_KEY=<your-openai-api-key>
+```
+
+Then, simply run the project in Dev mode:
+
+```
+mvn quarkus:dev
+```
+
+## Using the example
+
+Open your browser and navigate to http://localhost:8080. Click the red robot
+in the bottom right corner to open the chat window.
+
+The chatbot is a financial assistant that:
+1. Answers questions about financial products using information retrieved from documents
+2. Provides current stock prices for selected companies (AAPL, GOOG, MSFT)
+3. Analyzes sentiment in user messages
+4. Content moderation: Detects malicious content in user messages and sends a warning by email, if detected
+
+### Setting up the document catalog
+
+Before using the RAG functionality, you need to create a `catalog` directory with your financial product documents:
```

Comment:
This sounds like a prerequisite before running the app, but the app is configured to use the `catalog` directory that is under the root of the project, so this isn't actually needed, no? Rather we could just make a note that the app is configured to look for documents in a `catalog` directory relative to the current working directory (this will be useful in case people will want to run it in prod mode too)